### ISSUE TITLE
Add timestamps?

### ISSUE THREAD

**Issue by willuhmjs** [alias-rahil/lyrics-searcher#2]
**Is your feature request related to a problem? Please describe.**
The package works find when grabbing lyrics, but no timestamps are given.

**Describe the solution you'd like**
Can a feature be developed to return timestamps for lyrics?

**Comment by alias-rahil** [comment]
Yes I am open to any PR to improve/add functionality. We can possibly fetch other type of metadata (like album name or the movie it is from, etc) for the song too. Lmk if you want to take this up.
